### ISSUE TITLE
Add option to import cookies.txt

### ISSUE THREAD

**Issue by tonycpsu** [ysard/cookie-quick-manager#84]
The `README` says:

> Export: The export and import of a cookie or cookies from a domain in JSON or Netscape format is just as easy.

I see that export to text is supported, but I'm unable to find an option to import from text.  The import icon seems to provide only one option, and when I point it at a text file, the cookies are not imported.  If this feature isn't currently supported, it would be very useful for me.

**Comment by josephh** [comment]
Yes this is misleading: how do i import a cookies.txt file?  Is this a user error (on my part) or is the documentation wrong?
